README.rst

```diff
@@ -32,84 +32,168 @@ Features
 --------
 
 **Three layers of Python support for working with Apache NiFi:**
```

Comment:
Should this say: "- Top-level Api (see `examples/` in the repo)?
Also, if the examples had their own README, this would be better as a link the that

Reply:
Fixed in [add80b7](https://github.com/Chaffelson/nipyapi/pull/388/commits/add80b7f0838918a178fb285e9904cd31ac92ece)